incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/view/Resolvable.java

```diff
@@ -39,7 +41,7 @@ protected T resolveRef(
         Matcher matcher = pattern.matcher(ref);
```

Comment:
Hoist pattern (or matcher if reusable) to constructor.

Reply:
done